numpy/lib/shape_base.py

```diff
@@ -255,6 +255,7 @@ def expand_dims(a, axis):
 
     See Also
     --------
+    squeeze : The inverse operation, removing singleton dimensions
```

Comment:
What I mean here is that `np.squeeze(arr, axis=a)` is the inverse of `np.expand_dims(arr, axis=a)`. `squeeze`